source/deployment/ldap-group-sync.rst

```diff
@@ -20,9 +20,8 @@ The synchronization of groups happens with the synchronization of users, during
 Enabling AD/LDAP Group Synchronization
 ------------------------------------
 
-To enable this feature, go to **System Console > Experimental > Features** in versions after 5.12 (or **System Console > Advanced > Experimental > Enable AD/LDAP Group Sync** in prior versions).
+To enable this feature, go to **System Console > Groups** in versions after 5.12 (or **System Console > Advanced > Experimental > Enable AD/LDAP Group Sync** in prior versions).
```

Comment:
Confirm the steps in versions prior to 5.12 are correct.

Reply:
This is correct - 5.12 had the Experimental category in the System Console.